automated install/basic-install.sh

```diff
@@ -1432,14 +1432,23 @@ installConfigs() {
         elif [[ -d "/etc/lighttpd/conf-available" ]]; then
             conf=/etc/lighttpd/conf-available/15-pihole-admin.conf
             install -D -m 644 -T ${PI_HOLE_LOCAL_REPO}/advanced/pihole-admin.conf $conf
-            # disable server.modules += ( ... ) in $conf to avoid module dups
-            # (needed until Debian 10 no longer supported by pi-hole)
-            # (server.modules duplication is ignored in lighttpd 1.4.56+)
-            if awk '!/^server\.modules/{print}' $conf > $conf.$$ && mv $conf.$$ $conf; then
+
+            # Get the version number of lighttpd
+            version=$(dpkg-query -f='${Version}\n' --show lighttpd)
+            # Test if that version is greater than or euqal to 1.4.56
+            if dpkg --compare-versions "$version" "ge" "1.4.56"; then
+                # If it is, then we don't need to disable the modules
                 :
             else
-                rm $conf.$$
+                # disable server.modules += ( ... ) in $conf to avoid module dups
+                # (server.modules duplication is ignored in lighttpd 1.4.56+)
```

Comment:
```suggestion
                # (server.modules duplication is ignored in lighttpd <=1.4.56)
```

Reply:
I think you mean `>=`

Reply:
Ah yes I think we should be using `:=` on the following line

https://github.com/pi-hole/pi-hole/blob/9048429bbbc002c851365d3ccef1c5923ef8845c/advanced/pihole-admin.conf#L20

Do we even need to be overriding the access log file name? I don't think any of our scripts depend on it.  (Actually the same could be said for the error log - I don't recall why we prepended `pihole-` to their filenames in the first place... )

Reply:
Reason must be somewhere here: https://github.com/pi-hole/pi-hole/pull/3732

Reply:
> I think you mean >=

As we check `"$version" "ge" "1.4.56` and do nothing in that case, the removal happens when the version is smaller than 1.4.56

Reply:
Ah, I see what you mean. Looking back at it, this line would be better positioned in the do nothing case.

Reply:
@yubiuser - I've just FPd another change - can you try on an existing install again please?

Reply:
Though I've just tested this on a friends pi (shh, don't tell them) and it seems OK:

```
pi@raspberrypi:~ $ lighttpd -tt -f /etc/lighttpd/lighttpd.conf
2023-01-21 14:43:57: configfile.c.2274) server.upload-dirs doesn't exist: /var/cache/lighttpd/uploads
```

I think the `/var/cache/` notice is irrelevant in this case?

Reply:
I'll try later but think it won't work. The path in ` *.debian.conf` and `15-pihole-admin.conf` don't conflict, they are the same. But they conflict with `10-accesslog.conf` because the path is different.

Reply:
The paths that were conflicting were the one in `*.debian.conf` and `10-accesslog.conf`. The one in `15-pihole-admin.conf` is employed only when the request falls into the `$HTTP["url"] =~ "^/admin/" {` section of the config.

See https://redmine.lighttpd.net/projects/lighttpd/wiki/Mod_accesslog for an example where multiple acceslog paths are defined.

Like I said - I tested this and found it to be working OK - my only question is the `server.upload-dirs` part, but that is actually a false flag and nothing to do with this PR/issue

Reply:
Happy to be proven wrong.